keras/callbacks.py

```diff
@@ -1253,6 +1253,9 @@ class ModelCheckpoint(Callback):
       options: Optional `tf.train.CheckpointOptions` object if
         `save_weights_only` is true or optional `tf.saved_model.SaveOptions`
         object if `save_weights_only` is false.
+      initial_value_threshold: Initial "best" value of the metric to be monitored.
+        Only overwrites the model weights already saved if the
+        performance of current model is better than this value.
```

Comment:
This only applies when save_best_only is true correct? We should call that out. Something like (add newlines).

Initial "best" value of the metric to be monitored. Only applies if `save_best_value=True`. If set, the checkpoint will only be saved if the model metric value is better than this value.

Reply:
Thank you for pointing out that. Would you please check it again?

Reply:
Thank you for pointing that out. Could you please take a look again?